client/a8c-for-agencies/sections/sites/sites-notifications/jetpack-site-connected.tsx

```diff
@@ -0,0 +1,52 @@
+import page from '@automattic/calypso-router';
+import { getQueryArg, removeQueryArgs } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import { useContext, useEffect, useState } from 'react';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { successNotice } from 'calypso/state/notices/actions';
+import SitesDashboardContext from '../sites-dashboard-context';
+
+export default function JetpackSiteConnected() {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const jetpackConnectedSite = ( getQueryArg( window.location.href, 'site_connected' ) ??
+		'' ) as string;
+	const [ successNotification, setSuccessNotification ] = useState< React.ReactNode >( null );
+	const { setMostRecentConnectedSite } = useContext( SitesDashboardContext );
+	// Set the success notification when the site is connected and remove the query arg from the URL
+	useEffect( () => {
+		if ( jetpackConnectedSite ) {
+			setSuccessNotification( () =>
+				translate( '{{em}}%(jetpackConnectedSite)s{{/em}} was successfully connected to Jetpack', {
```

Comment:
In the description, you say it no longer mentions Jetpack, but this line proves the opposite. Could it be that you forgot to push a commit?

Reply:
Good catch, thanks! Fixed via [9746286](https://github.com/Automattic/wp-calypso/pull/88736/commits/9746286981ceb1402275f556e613b27048f2c9b6).